frontend/pages/hosts/details/cards/Activity/PastCommandFeed/PastCommandFeed.tsx

```diff
@@ -45,13 +47,13 @@ const PastCommandFeed = ({
           );
         })}
       </div>
-      {/* <Pagination */}
-      {/*   disablePrev={!meta.has_previous_results} */}
-      {/*   disableNext={!meta.has_next_results} */}
-      {/*   hidePagination={!meta.has_next_results && !meta.has_previous_results} */}
-      {/*   onPrevPage={onPreviousPage} */}
-      {/*   onNextPage={onNextPage} */}
-      {/* /> */}
+      <Pagination
```

Comment:
Pagination seems to be shared between the toggle of MDM commands and regular activities.

I can paginate a lot on the MDM commands, then flip the switch to see no results, however I know there is if I reset the pagination.